packages/go.sh

```diff
@@ -9,10 +9,16 @@ if [[ `go version` = "go version go$VERSION "* ]]; then
     exit
 fi
 
-start_install
-  wget https://dl.google.com/go/go$VERSION.linux-amd64.tar.gz
-  tar xzvf go$VERSION.linux-amd64.tar.gz
-  sudo mv go /usr/local/go-$VERSION
-  sudo ln -sf --no-dereference /usr/local/go-$VERSION /usr/local/go
-  rm go$VERSION.linux-amd64.tar.gz
-end_install
+if [ "$(uname)" == "Darwin" ]; then
+brew bundle --file=- <<-EOS
+  brew "go@1.12"
```

Comment:
can this use the $VERSION variable?

Reply:
It needs only major and minor -_- I tried it with patch

Reply:
(and I was too lazy to clip it....)